### PR TITLE
Connect Forgot password to backend

### PR DESCRIPTION
**Pull request by anandangalig**
Also, removed .env file.

### CODE REVIEW

File: Components/Molecules/SignUpContainer.js [hunk 1]
```diff
@@ -34,6 +35,13 @@ const SignUpContainer = ({
             <View>{children}</View>
             <Button label={nextLabel || "Next"} onPress={next} full primary />
             <Button label="Back" onPress={back} full />
+            <Button
```
Comment (ChrisCruze) [inline Components/Molecules/SignUpContainer.js:38]: The SignUpContainer is used by the SignUpEmail, SignUpName, and SignUpPassword pages, which means Forgot Password is showing up  there. 
Screenshot 2020-05-17 16 17 15

Comment (anandangalig) [reply Components/Molecules/SignUpContainer.js]: good catch, added a condition check to render this button only on Login screen

File: Components/Molecules/SignUpContainer.js [hunk 2]
```diff
@@ -34,6 +35,15 @@ const SignUpContainer = ({
             <View>{children}</View>
             <Button label={nextLabel || "Next"} onPress={next} full primary />
             <Button label="Back" onPress={back} full />
+            {navigation.state.routeName === "Login" && (
```
Comment (ChrisCruze) [inline Components/Molecules/SignUpContainer.js:38]: nice solution - didnt realize you can do that

**Comment by ChrisCruze** [comment]
I'm merging this in so I can start the build for our next release

**Pull request merged**